src/vector_transport.jl

```diff
@@ -252,15 +252,27 @@ struct SchildsLadderTransport{
     end
 end
 
+"""
+    default_vector_transport_method(M::AbstractManifold)
+
+The [`AbstractVectorTransportMethod`](@ref) that is used when calling
+[`vector_transport_along`](@ref), [`vector_transport_to`](@ref), or
+[`vector_transport_direction`](@ref) without specifying the vector transport method.
+By default, this is `DifferentiatedRetractionVectorTransport(default_retraction_method(M))`.
+"""
+function default_vector_transport_method(M::AbstractManifold)
+    return DifferentiatedRetractionVectorTransport(default_retraction_method(M))
+end
+
 @doc raw"""
     pole_ladder(
         M,
         p,
         d,
         q,
         c = mid_point(M, p, q);
-        retraction=ExponentialRetraction(),
-        inverse_retraction=LogarithmicInverseRetraction()
+        retraction=default_retraction_method(M),
```

Comment:
Merely a comment – while I was not 100% sure for the default with retract (one could jus also dispatch the verity without a retraction method to a new/different default for a manifold) what I really like is how this turned out! Here the new function yields a really nice feature!